### DISCUSSION TITLE
Recommended way to import JSON data

### DISCUSSION THREAD

**Discussion by dannyvalle** [nuxt/nuxt#18109] [Questions] [answered]
With NUXT 3, I have a products page and I want to store the data in a JSON file. I'd like to have a top level page to list all products and then single product pages.

Where is the best place to store this file in NUXT 3 — static folder and use "import" or server/api folder and use "useFetch()"?

**Comment by wobsoriano** [comment, marked as answer]
I always put my json data files in a `data` folder then import them like this:

```ts
// server/api/todos.ts
import todos from "~/data/todos.json";

export default (req, res) => {
  return todos;
};
```

**Reply by dannyvalle** [reply]
Right. I did try this at one point, the issue I had was I can’t use “pick” when using useFetch(). It returns an empty object.

**Reply by wobsoriano** [reply]
This doesn't work?

```ts
const { data } = await useFetch('/api/todos')
```

**Reply by dannyvalle** [reply]
That does work, it was when I added the "pick" option, but I just realized what I was doing wrong. I had the following:

`const { data } = await useFetch('/api/todos', {pick: "key_name" })`

instead of:

`const { data } = await useFetch('/api/todos', {pick: ["key_name"] })`

I had key_name wrapped in double quotes without brackets. Now all works. Thank you.
